dedupe/manifests/record-linkage.yml

```diff
@@ -0,0 +1,154 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: postgres-db-config
+  labels:
+    app: postgresql-db
+data:
+  POSTGRES_DB: testdb
+  POSTGRES_USER: postgres
+  POSTGRES_PASSWORD: pw
+  PGUSER: postgres
+  PGDATA: /data/pgdata
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: postgres-db-lb
+spec:
+  selector:
+    app: postgresql-db
+  type: LoadBalancer
```

Comment:
I'm a bit new to kubernetes, but is the database loadbalancer just needed to forward the postgres service to the host?

Reply:
Correct! So in k8s there are 3 options to choose from, ClusterIP - which is only internal to the cluster and allows automatic communication between resources(also, ClusterIP is enabled during the other two types as well, for good reason), and then theres NodePort and Loadbalancer. They both essentially do the same things which is connecting with the Node to create an external IP. The main difference is that Loadbalancer is the best option for cloud provider connectivity, to then do other things from there like dns, certificates, Ingress and egress, etc. tbh we probably should just to NodePort here because this is just a local sandbox environment anyways but I'm trying to keep it as close to the real thing as possible. 

So once any of those are set. There are certain ways to communicate with them from CLI, without getting too into that, to summarize, if we were to use ClusterIP we could need to do whats called port-fowarding for each service, every time we want to them. For NodePort and Loadbalancer, it depends on the provider/platform your are using but in our case we are using Minikube, in the script we use the command `minikube tunnel` and that will expose an external IP for you to use when running `kubectl get all`. If you dont use the tunnel the external-IP will show up as `<Pending>`.

Sorry for the brain dump on Kube lol